packages/draft-js-import-markdown/src/MarkdownParser.js

```diff
@@ -31,6 +31,7 @@ noop.exec = noop;
 
 var defaults = {
   gfm: true,
+  headers: true,
```

Comment:
Maybe rename to `headings`?  `headers` is a little ambiguous in this context.

Reply:
Sure